### DISCUSSION TITLE
Disable Max Lens crop when stabilisation is off

### DISCUSSION THREAD

**Discussion by IRCGraphic** [gopro/labs#302] [Feature Requests]
When in Max lens mod camera mode and the stabilisation is off it maintains the low res digital zoom loosing pixel density, more noise and pixel artefacts, etc, could the full sensor crop be disabled when stabilisation is off in the Max lens mode?

**Comment by dnewman-gpsw** [comment]
Not something Labs can change.  However you you want all the pixels, just add the Max Lens Mod, with the feature disabled (in camera menu), then shoot HS off, 5.3K 8:7, than you get every pixel.  If you stabilizing in post, you may also want to set oMBERS=1.

  
![image](https://user-images.githubusercontent.com/10856084/230992104-8c80cff9-8cc8-4e79-a1a7-efa6255b2098.png)

**Comment by IRCGraphic** [comment]
I see BypassElectronicShutter is exclusive for GoPro 10 and up, i film with a GoPro 9 and yes it is for post stabilising  -people been asking for Max lens post stabilisation since its release in the GoPro 9

**Reply by dnewman-gpsw** [reply]
Unfortunately it took a GP2 processor to work out how to do BERS (Bypass ERS, is not full EIS.)  There is no opportunity to back port this one, I've tried. For technical reasons and engineering/QA bandwidth, this feature will not happen on HERO9.

**Comment by IRCGraphic** [comment]
Yeah Labs been braking grounds, in pair with GoPro's ingenuity, that's why it always looked really strange to make that heavy crop to the Max lens without stabilisation, wouldn't that be really bad for Reelsteady, still incompatible with the fisheye mod last time I checked, and for the novelty of the Max lens? Nevermind, I guess then I should be addressing the main GoPro forum.
Thanks for your support.
